app/models/ems_refresh/save_inventory_helper.rb

```diff
@@ -68,7 +68,7 @@ def save_inventory_multi(association, hashes, deletes, find_key, child_keys = []
   def save_inventory_single(type, parent, hash, child_keys = [], extra_keys = [], disconnect = false)
     child = parent.send(type)
     if hash.blank?
-      disconnect ? child.disconnect_inv : child.try(:destroy)
+      disconnect ? child.try(:disconnect_inv) : child.try(:destroy)
```

Comment:
> This could be a valid alternative.

@simon3z  @zeari https://github.com/ManageIQ/manageiq/pull/8329 fixes the underlying issue that may cause problems in many places, here we only deal with one such place.